lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -16,9 +16,6 @@ const DOM = require('../../../../report/html/renderer/dom.js');
 const DetailsRenderer = require('../../../../report/html/renderer/details-renderer.js');
 const ReportUIFeatures = require('../../../../report/html/renderer/report-ui-features.js');
 const CategoryRenderer = require('../../../../report/html/renderer/category-renderer.js');
-// lazy loaded because they depend on CategoryRenderer to be available globally
```

Comment:
these last two files cleaned up based on @patrickhulce feedback in https://github.com/GoogleChrome/lighthouse/pull/6486#discussion_r231355076